client/rpc/status.go

```diff
@@ -53,12 +52,7 @@ func printNodeStatus(_ *cobra.Command, _ []string) error {
 		return err
 	}
 
-	var output []byte
-	if clientCtx.Indent {
```

Comment:
Why is all the indent stuff going away?

Reply:
Pointless technical debt to maintain. It should have never been introduced in the first place.